frontend/src/metabase-types/store/dashboard.ts

```diff
@@ -10,16 +10,29 @@ import { ParameterValueOrArray } from "metabase-types/types/Parameter";
 
 export type DashboardSidebarName =
   | "addQuestion"
+  | "action"
   | "clickBehavior"
   | "editParameter"
   | "sharing"
   | "info";
 
+export type StoreDashboard = Omit<Dashboard, "ordered_cards"> & {
```

Comment:
Improving the types here to match what's actually in the Redux store to prevent type errors.